### PR TITLE
Xsoar playbook fixes

### PR DESCRIPTION
**Pull request by npatellumeta**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by dansterenson** [comment]
@npatellumet please register your contribution by filling the [contribution registration form ](https://forms.gle/XDfxU4E61ZwEESSMA).

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: Packs/FireMonSecurityManager/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "FireMon Security Manager",
     "description": "Creates a Policy Planner Ticket in Policy Planner Application and Validates Pre Changes assessments for new requirement.",
     "support": "partner",
-    "currentVersion": "1.0.1",
+    "currentVersion": "1.0.2",
```
[the remark this answers is not in the document]
Comment (npatellumeta) [reply Packs/FireMonSecurityManager/pack_metadata.json]: We have applied this change and pushed it to git

**Review by dansterenson — APPROVED** [review]

File: Packs/FireMonSecurityManager/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Incident Types
+- **FireMon Pre Change Assessment**
+- **FireMon Policy Planner**
+
+#### Integrations
+##### FireMon Security Manager
+- Fixing playbook Id
```
Comment (dansterenson) [inline Packs/FireMonSecurityManager/ReleaseNotes/1_0_2.md:8]:
```suggestion
#### Incident Types
##### FireMon Policy Planner
 - Fixed Layout name.
##### FireMon Pre Change Assessment
 - Fixed Layout name.
```

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: Packs/FireMonSecurityManager/ReleaseNotes/1_0_2.md (hunk 2 above)
Comment (npatellumeta) [reply Packs/FireMonSecurityManager/ReleaseNotes/1_0_2.md:8]: If these are mere recommendation, i would like to skip making these changes. I exactly followed what the instructions  said at the command line while running command to generate readme file. I am extremely sorry but this is just getting a bit ridiculous.

**Review by dansterenson — APPROVED** [review]

**Pull request merged**